security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_generate_conf.php

```diff
@@ -338,7 +338,7 @@
 				    "telnet_cmds" => "no", "ignore_telnet_erase_cmds" => "yes", 
 				    "bounce" => "yes", "bounce_to_net" => "", "bounce_to_port" => "" );
 
-if (!is_array($snortcfg['ftp_client_engine']['item'])) {
+if (!array_get_path($snortcfg, 'ftp_client_engine/item')) {
 	$snortcfg['ftp_client_engine'] = array();
 	$snortcfg['ftp_client_engine']['item'] = array();
```

Comment:
```suggestion
array_init_path($snortcfg, 'ftp_client_engine/item');
```

Reply:
Thanks for the tip. Makes the code simpler and easier to read.